backend/apps/notifications/tasks.py

```
@@ -247,29 +258,64 @@ def send_notification_digests():
+    # so no broker is required; otherwise dispatch via `apply_async`.
+    eager = getattr(settings, "CELERY_TASK_ALWAYS_EAGER", False)
+    for batch in chunks:
+        if not batch:
+            continue
         try:
-            send_bulk_email(payload)
+            if eager:
+                group(batch).apply()
+            else:
+                group(batch).apply_async()
         except Exception as exc:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)